localgov_news.module

```diff
@@ -86,7 +86,9 @@ function localgov_news_field_widget_complete_form_alter(&$field_widget_complete_
     if (count($options) == 1 && isset($options['_none'])) {
       $create_newsroom_path = 'localgov_newsroom';
       if (\Drupal::moduleHandler()->moduleExists('localgov_microsites_group')) {
+        // @phpstan-ignore-next-line
```

Comment:
Does it make sense to comment why the line is ignored?

Reply:
Hi @ekes I've added a comment to both lines